### DISCUSSION TITLE
What is the default hash size?

### DISCUSSION THREAD

**Discussion by Mercutio1243** [official-stockfish/Stockfish#4996] [Q&A] [answered]
I have searched on this topic but while I (hopefully) understand the concept of hash and the hashful information that stockfish can provide, I seemingly cannot find an answer to the folloeing question:

Let's say I just initialize Stockfish without specifying the hash size and give it a position to evaluate. Which default hash size will it use? Does the hash size depend on the device?

**Comment by Craftyawesome** [comment, marked as answer]
16 MiB. You can also see the defaults of all options typing `uci`

**Reply by Mercutio1243** [reply]
Thank you! 

Let's say I analyze a few separate positions, possibly wanting to go deeper into one of the positiony once another has been analysed by Stockfish.

From experience, what would be an appropriate hash size for this situation? 

Also, is there any way to clear the hash, once I know that I will not return to analyzing a specific position?

**Reply by Craftyawesome** [reply]
Try to keep hashfull under 30%. But there is very little penalty for too large hash as long as your computer has the memory for it.

You probably don't need to clear the hash since old entries will just get replaced, but if you need to there is a "Clear Hash" button.

**Comment by Disservin** [comment]
Also have a look at https://github.com/official-stockfish/Stockfish/wiki/Stockfish-FAQ#optimal-settings

**Reply by Mercutio1243** [reply]
Thank you!

For quick reference for future readers, also refer to: https://github.com/official-stockfish/Stockfish/wiki/UCI-&-Commands
